.rubocop.yml

```diff
@@ -35,8 +32,8 @@ Naming/FileName:
 Naming/HeredocDelimiterNaming:
   Enabled: false
 
-Style/TrailingCommaInLiteral:
-  Enabled: false
+# Style/TrailingCommaInLiteral:
```

Comment:
Please drop the commented lines

Reply:
Updated this changes here https://github.com/theforeman/smart_proxy_dynflow/pull/118

I have moved Rubocop and jenkins work to #118.